src/components/auth-provider.tsx

```diff
@@ -181,8 +176,6 @@ function createWpcomClient(
 		}
 	};
 
-	const originalRequestHandler = wpcom.request.bind( wpcom );
-
 	const addLocaleToParams = ( params: WpcomParams ) => {
```

Comment:
```suggestion
	const addLocaleToParams = ( params: object ) => {
```
Seems like we can just use the `object` type here.

Reply:
It seems the type was added intentionally in https://github.com/Automattic/studio/commit/1f8cb5983ec9fee912cb45aafdaa8331d09d330e, so I would leave it.